components/ActionSheet/src/MDCActionSheetController.m

```diff
@@ -175,7 +179,14 @@ - (void)viewWillLayoutSubviews {
   }
   CGSize size = [self.header sizeThatFits:CGRectStandardize(self.view.bounds).size];
   self.header.frame = CGRectMake(0, 0, self.view.bounds.size.width, size.height);
-  UIEdgeInsets insets = UIEdgeInsetsMake(self.header.frame.size.height, 0, 0, 0);
+  if (self.title != nil || self.message != nil) {
```

Comment:
Shouldn't this be checking the `headerDividerColor` to see if it's `nil`? As I understand the header documentation, it's the color that determines if the view is added.

Reply:
This has been updated the headerDividerView is never nil.